beacon-chain/rpc/eth/beacon/handlers_pool.go

```diff
@@ -159,13 +165,19 @@ func (s *Server) SubmitAttestations(w http.ResponseWriter, r *http.Request) {
 			log.WithError(err).Errorf("could not broadcast attestation at index %d", i)
 		}
 
-		if corehelpers.IsAggregated(att) {
-			if err = s.AttestationsPool.SaveAggregatedAttestation(att); err != nil {
-				log.WithError(err).Error("could not save aggregated attestation")
+		if features.Get().EnableExperimentalAttestationPool {
+			if err = s.AttestationCache.Add(att); err != nil {
+				log.WithError(err).Error("could not save attestation")
 			}
 		} else {
-			if err = s.AttestationsPool.SaveUnaggregatedAttestation(att); err != nil {
-				log.WithError(err).Error("could not save unaggregated attestation")
+			if att.IsAggregated() {
```

Comment:
Same as in previous comments, a level of tabs can be removed here by using else if